src/open_inwoner/components/templates/components/Form/Form.html

```diff
@@ -27,7 +27,7 @@
     {% endif %}
 
     {% if show_required %}
-        <span class="caption__content" id="caption">{% trans "Velden gemarkeerd met een" %}<span class="label__label--required"> * </span>{% trans "zijn verplicht" %}</span>
+        <span class="caption__content" id="caption">{% trans "Velden gemarkeerd met een sterretje (" %}<span class="label__label--required">*</span>{% trans ") zijn verplicht" %}</span>
```

Comment:
The parentheses can be out of the translated text.

Reply:
It is not good to break a single line into two translatables, because the parts make no sense on their own and lack context (also order of replacements could be language specific). Use a `blocktrans` with a `with` attribute (and a `|safe` filter to allow html):

````
{% blocktrans trimmed with star='<span class="label__label--required"> * </span>'|safe %}
Velden gemarkeerd met een {{star}} zijn verplicht
{% endblocktrans%}
````

Reply:
Thanks @Bartvaderkin I will change it into `blocktrans` (even though you approved it already :-D )